core/src/it/scala/org/ensime/fixture/SearchServiceFixture.scala

```diff
@@ -12,7 +13,7 @@ trait IsolatedSearchServiceFixture extends IsolatedSourceResolverFixture {
     try {
       testCode(config, searchService)
     } finally {
-      searchService.shutdown()
+      Await.ready(searchService.shutdown(), 10.seconds)
```

Comment:
I have no idea, I just picked 10 seconds because of the 10 seconds below and though that Slick should reasonably shut down much faster than that. Tell me if a change is needed.